libraries/AP_Arming/AP_Arming.h

```diff
@@ -68,6 +68,8 @@ class AP_Arming {
         TAKEOFFTIMEOUT = 27, // only disarm uses this...
         AUTOLANDED = 28, // only disarm uses this...
         PILOT_INPUT_FAILSAFE = 29, // only disarm uses this...
+        TLANDTHROTTLE = 30, // only disarm uses this...
```

Comment:
Can we rename this to include the word TOY?

Reply:
Done.